docs/package.json

```
@@ -20,7 +20,11 @@
     "remove-version": "node --unhandled-rejections=strict ./scripts/remove-version.js"
   },
   "resolutions": {
-    "webpack": "^5.28.0"
+    "@babel/core": "^7.17.10",
+    "@babel/helper-plugin-utils": "^7.16.7",
+    "@babel/plugin-syntax-jsx": "^7.16.7",
+    "@babel/plugin-syntax-object-rest-spread": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.17.3"
```

```suggestion
    "@babel/core": "^7.17.10",
    "@babel/helper-plugin-utils": "^7.16.7",
    "@babel/plugin-syntax-jsx": "^7.16.7",
    "@babel/plugin-syntax-object-rest-spread": "^7.8.3",
    "@babel/plugin-proposal-object-rest-spread": "^7.17.3"
```

I'm not sure if those resolution are required, they will downgrade the Babel we use.

---

Removing them! This was in Cedrics original PR and I did not clean them up. Thanks for keeping an eye out